pom.xml

```diff
@@ -285,7 +285,7 @@
       <dependency>
         <groupId>org.apache.maven.shared</groupId>
         <artifactId>maven-common-artifact-filters</artifactId>
-        <version>3.3.2</version>
+        <version>3.4.0</version>
```

Comment:
unsure here

Reply:
That one has no OSGi manifest, so it can't hurt to upgrade.